src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator start working!");
```

Comment:
Consider making the start message more informative. For example: 'Excavator has begun its work.' This follows the checklist item of writing informative messages in methods.